### PR TITLE
Custom Serve and Error Handler for Services

### PR DESCRIPTION
**Pull request by MrToino**
The purpose of this component is to implement a custom MPB server handler, to be used on all services. Some error handlers will also be included in this custom server.

### CODE REVIEW

File: intmodules/log.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import logging.config
 
 
-log_path = "../services/logs\\"
+log_path = "..\\services\\logs\\"
```
Comment (luisferreira32) [inline intmodules/log.py:6]: Don't do this, windows vs unix systems will break. use a path join or sth like that

File: intmodules/__init__.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-from intmodules.log_module import config_logger
-from intmodules.log_module import logger
+from intmodules.log import config_logger
+from intmodules.log import logger
```
Comment (luisferreira32) [inline intmodules/__init__.py:2]:
```suggestion
from intmodules.log import config_logger, logger
```

**Review by luisferreira32 — CHANGES_REQUESTED** [review]

File: intmodules/__init__.py (hunk 2 above)
Comment (MrToino) [reply intmodules/__init__.py:2]: You are right. I was following some good practices (https://aglowiditsolutions.com/blog/python-best-practices/). 
In fact, having multiple "from ____ import ____" it is okay to put it all in one same line. Having multiple "import ____" we must leave them in different lines. I will make the change, in this case.

**Comment by MrToino** [comment]
Server handler related changes will be implemented on another [PR](https://github.com/MrToino/MPB/pull/17). 
This pull request will await the completeness of the prior request.

**Review by luisferreira32 — APPROVED** [review]
Once again this PR files changed are all broken... another way to try to avoid this could also be reset all commits (soft reset), stash them, rebase from main, then unstash, solve conflicts and voilá create your own new commit 👍

**Pull request merged**